src/net/network_config.rs

```diff
@@ -1,21 +1,39 @@
 use std::default::Default;
+use net::constants::{FRAGMENT_SIZE_DEFAULT, MAX_FRAGMENTS_DEFAULT,};
+
 
 #[derive(Clone)]
 pub struct NetworkConfig
 {
+    /// This is the maximal size an packet can get with all its fragments.
+    ///
+    /// Recommended value: 16384
     pub max_packet_size: usize,
+    /// These are the maximal fragments a packet could be divided into.
+    ///
+    /// Why can't I have more than 256 (u8)?
+    /// This is because you don't want to send more then 256 fragments over UDP, with high amounts of fragments the change for an invalid packet is very high.
```

Comment:
"change" -> "chance"

Dumb question from somebody who doesn't know much about networking: If one packet failed, couldn't we just resend it? I think it would be ideal if the size of the packet can be as large as necessary, so the user does not have to split very big messages manually.

Reply:
One time stuff? What we want to have like TCP: ordered +  able to see dropped packets (not automatically resent them) + the fragmentation (this PR, and later to be able to send each fragment individually if dropped).

Reply:
@torkleyy You can also think of it this way: TCP is built on top of UDP. We're doing the same thing, but only implementing the few features of TCP that are useful to a network game.

Regarding packet size, when sending data over the Internet, in general you want to keep the packet size at less than 1500 bytes. The reason for this is historical. Thus, we have to implement our own fragmentation, since plain UDP does not have it.

Reply:
@TimonPost I meant things like assets. I'm just reading all the documents and @fhaynes answers my questions on Discord ;) I understood this now, you can see this as resolved except for "change" which should be "chance".

Reply:
I'll have already changed that thanks for looking into the PR